data-model/fdo-type/source-system/0.3.0/schema/source-system.json

```diff
@@ -61,20 +69,10 @@
         "2021-06-01T12:00:000Z"
       ]
     },
-    "ods:dateTombstoned": {
+    "schema:creator": {
       "type": "string",
```

Comment:
can we use the `schema` namespace and then point to an `ods` object (agent)? This doesn't seem to me like it's an instance of `schema:creator`

Reply:
Yes in the instance of `schema` we can actually do it as they leave it open-ended. In addition, the type of the object is not from the `ods` vocab. I you look in the agent you will see that we have five types defined there (where the `prov` types are only used in the provenance object).